Readme.md

```diff
@@ -0,0 +1,72 @@
+# Intellij-Couchbase README
+
+Welcome to the official Intellij Idea extension for Couchbase!
+
+This plugin is designed to provide a seamless experience for Couchbase Server users who want to work within the popular Intellij Idea editor.
+
+# Requirements
+This extension requires OpenSSL to be installed on your system in order to install successfully. 
+
```

Comment:
Can you elaborate on why openssl is a requirement for this plugin? Feeld like you copy and pasted it here without putting too much effort on thinking if this is something that should be added or not.